packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -158,6 +158,12 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
     if (ev.which === KeyCodes.enter) {
       ev.preventDefault();
       this._onGotoToday();
+    } else if (ev.which === KeyCodes.tab && !ev.shiftKey) {
```

Comment:
You should add similar logic for `shift+tab` on the first tab stop of the Calendar